lib/active_triples/relation.rb

```diff
@@ -233,13 +254,13 @@ def class_for_property
         klass
       end
 
+      ##
+      # @return [RDF::Term] the subject of the relation
       def rdf_subject
-        raise ArgumentError, "wrong number of arguments (#{value_arguments.length} for 1-2)" if value_arguments.length < 1 || value_arguments.length > 2
-        if value_arguments.length > 1
-          value_arguments.first
-        else
-          parent.rdf_subject
-        end
+        raise ArgumentError, 'wrong number of arguments' \
```

Comment:
The refactor on `raise` is just to wrap the line at <= 80 chars.

The ternary operator seemed better, per: https://github.com/bbatsov/ruby-style-guide/blob/master/README.md#ternary-operator

I'm not a big fan of the long `raise` line, anyway, but don't have better ideas. Maybe just an empty line at 263?